ESP8266/EzTimeTetrisClock_ssd1306OLED/EzTimeTetrisClock_ssd1306OLED.ino

```diff
@@ -0,0 +1,321 @@
+  tetris.drawChar("n", x + 42, y, WHITE);
+  tetris.drawChar("g", x + 47, y, WHITE);
+}
+
+void setup() {
+  // Serial.begin(115200);
+
+  // Attempt to connect to Wifi network:
+  Serial.print("Connecting Wifi: ");
+  Serial.println(ssid);
+
+  // Set WiFi to station mode and disconnect from an AP if it was Previously
+  // connected
+  
+  // WiFi.mode(WIFI_OFF);
+  WiFi.begin(ssid, password);
+  
+  if(!display.begin(SSD1306_SWITCHCAPVCC, 0x3C)) { // Address 0x3C for 64x48
+    Serial.println(F("SSD1306 allocation failed"));
+  }
+
+  display.clearDisplay();
+  display.setTextSize(1);             // Normal 1:1 pixel scale
+  display.setTextColor(WHITE);        // Draw white text
+  display.setCursor(0,0);             // Start at top-left corner
+  display.println(F("Connecting..."));
+  display.display();
+
+  delay(2000);
+  // Setup EZ Time
+  setDebug(INFO);
+  if(getTime){
+    if(WiFi.status()==WL_CONNECTED) waitForSync();
+
+    Serial.println();
+    Serial.println("UTC:             " + UTC.dateTime());
+
+    myTZ.setLocation(F(MYTIMEZONE));
+    Serial.print(F("Time in your set timezone:         "));
+    Serial.println(myTZ.dateTime());
+  }
+
+  display.clearDisplay();
+
+  if(displayIntro){
+    // tetris.scale = 1;
+    drawIntro(6, 12);
+    display.display();
+    delay(2000);
+    tetris.setText("B. LOUGH");
+  }
+
+  tetris.scale = scale;
+  setupMonochrome();
+}
+
+void setMatrixTime() {
+  Serial.println("setMatrixTime");
+  String timeString = "";
+  String AmPmString = "";
+
+  if (twelveHourFormat) {
+    // Get the time in format "1:15" or 11:15 (12 hour, no leading 0)
+    // Check the EZTime Github page for info on
+    // time formatting
+    timeString = myTZ.dateTime("g:i");
+
+    //If the length is only 4, pad it with
+    // a space at the beginning
+    if (timeString.length() == 4) {
+      timeString = " " + timeString;
+    }
+
+    //Get if its "AM" or "PM"
+    AmPmString = myTZ.dateTime("A");
+    if (lastDisplayedAmPm != AmPmString) {
+      Serial.println(AmPmString);
+      lastDisplayedAmPm = AmPmString;
+      // Second character is always "M"
+      // so need to parse it out
+      tetris2.setText("M", forceRefresh);
+
+      // Parse out first letter of String
+      // tetris3.setText(getTime ? AmPmString.substring(0, 1) : "A", forceRefresh);
+      tetris3.setText("A", forceRefresh);
+    }
+  } else {
+    // Get time in format "01:15" or "22:15"(24 hour with leading 0)
+    timeString = myTZ.dateTime("H:i");
+  }
+
+  // Only update Time if its different
+  if (lastDisplayedTime != timeString) {
+    // Serial.println(timeString);
+    lastDisplayedTime = timeString;
+    tetris.setTime(timeString, forceRefresh);
+    // Must set this to false so animation knows
+    // to start again
+    finishedAnimating = false;
+  }
+}
+
+void setupMonochrome(){
+  // @todo setup tetris colors
+  // 
+  // you can adjust these for different block appearance, colon not supported in library atm
```

Comment:
Hmm, good catch, bug for me, will add an issue to the library

Reply:
Whoops, you already did :)

https://github.com/toblum/TetrisAnimation/issues/5